src/App.tsx

```diff
@@ -1,10 +1,14 @@
-import './App.css';
-import ApplicationPage from './pages/applicationPage';
+import "./App.css";
+import ApplicationPage from "./pages/applicationPage";
 
 function App() {
   return (
     <div className="App">
-      <ApplicationPage requestName="Dummy Team" topLevelQuestion="Contact Info"/>
+      <ApplicationPage
+        requestName="Dummy Team"
+        topLevelQuestion="Contact Info"
+        privacyPolicyText=""
```

Comment:
Hmm, I have a bit of doubts passing this long string as a prop. Let's discuss this further, whether it would be better to load it directly from a static file or from an uploaded pdf or pass it as a prop :)

Reply:
Ah I see, alright let's discuss some later time! We can also fetch the data through API calls as well, or just use a static file if they are always the same.